### PR TITLE
Update forms.py

### PR DESCRIPTION
**Pull request by paubric**
Every form will indicate the page number of the specific question with dummy data.

### CODE REVIEW

File: project_template/forms.py [hunk 1]
```diff
@@ -5,94 +5,134 @@
 from project_template.datamodels.real_estate_type import RealEstateType
 from project_template.datamodels.attainment_type import AttainmentType
 
-YEAR_CHOICES = ('2008', '2009', '2010', '2011', '2012', '2013', '2014', '2015', '2016', '2017', '2018', '2019')
+YEAR_CHOICES = ('2008', '2009', '2010', '2011', '2012', '2013',
+                '2014', '2015', '2016', '2017', '2018', '2019')
+
+
+def get_table_of_contents():
+    return {"buildings": 1, "automobiles": 1, "jewelry": 2, "bank_accounts": 2, "investments": 2, "land": 1, "debts": 3, "goods": 3, "gifts_spouse": 3, "gifts_kids": 3, "salaries": 4, "independent_activities": 4, "income_investments": 4, "pensions": 4, "agriculture": 4, "other_sources": 5}
```
Comment (KovaxG) [inline project_template/forms.py]: this line is not like the others :)
poate s-ar putea scurta un pic

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (RaduCStefanescu) [inline .vscode/launch.json]: This file should not be part of the pull request.

File: project_template/task_templates.py [hunk 3]
```diff
@@ -38,5 +38,6 @@ def save_verified_data(self, verified_data):
 	def after_save(self, verified_data):
 		# Create a new task for each table, asking the user to transcribe the number of rows
 		number_rows = int(verified_data['count'])
-		for row_number in list(range(1, number_rows)):
+		for row_number in range(0, number_rows):
+			print("!!!!!!!!!!!!!!!!!!!!!!!!")
```
Comment (RaduCStefanescu) [inline project_template/task_templates.py]: This line should be removed.

**Comment by catileptic** [comment]
The overall purpose of this pull request was to implement a feature. That feature says that the pdf viewer should scroll to the page where the current table of interest is located. 

If I am a person transcribing asset declarations, and I am served a task which asks me to transcribe the second row of table 7, then the feature would scroll the pdf and show me the page where table 7 is listed. 

I see two major issues with this pull request that prevents it being okay to merge:
1. the feature implementation is incomplete - there is no code in place that "tells" the pdf viewer what page it should scroll to;
2. the pull requests also affects a whole lot of other files that could have gone untouched. 

My suggestion is for this feature to be left as a nice-to-have feature. If we have a working asset declaration digitalisation app and we have time to spare before going live with it, we will prioritise a feature such as this one, since it would definitely be useful.
